src/Tasks.php

```diff
@@ -1036,27 +1042,38 @@ private function downloadPantheonBackup($env) {
   }
 
   /**
-   * Imports a local database.
+   * Imports a local database using mysql rather than zcat.
    *
    * @return bool
    *   True if import succeeded.
    */
   public function importLocal() {
+    if (!$this->taskExec('which pv')->run()->wasSuccessful()) {
+      $this->yell("This project's database can take up to an hour to import. So that you can be informed of its progress, please install the pv tool with 'brew install pv', then run your command again.");
+      return FALSE;
+    }
+
     $project_properties = $this->getProjectProperties();
 
+    $web_root = $project_properties['web_root'];
+    $sql_db = $project_properties['db-name'];
+    $sql_user = $project_properties['db-user'];
+    $sql_pass = $project_properties['db-pass'];
     // Empty out the old database so deleted tables don't stick around.
-    $this->taskExec('drush sql:drop -y')->dir($project_properties['web_root'])->run();
-
-    $drush_commands    = [
-      'drush_import_database' => 'zcat < ../vendor/database.sql.gz | drush @self sqlc # Importing local copy of db.'
+    $this->taskExec('drush sql:drop -y')->dir($web_root)->run();
+    if (file_exists('vendor/database.sql.gz')) {
+      $this->taskExec('gunzip ../vendor/database.sql.gz')->dir($web_root)->run();
+    }
+    $import_commands    = [
+      'drush_import_database' => "pv ../vendor/database.sql | mysql -u$sql_user -p$sql_pass $sql_db # Importing local copy of db."
```

Comment:
You can get this string by running [`drush sql:connect` ](https://www.drush.org/latest/commands/sql_connect/)too, so:
```
      'drush_import_database' => "pv ../vendor/database.sql | $(drush sql:connect) # Importing local copy of db."
```

That also avoids printing out your credentials into the screen.

Reply:
done!